cmd/subctl/export.go

```diff
@@ -54,6 +54,21 @@ var (
 				}, cli.NewReporter()))
 		},
 	}
+	exportNamespaceCmd = &cobra.Command{
+		Use:   "namespace <namespaceName>",
```

Comment:
This indicates to me that the command exports an actual namespace resource. How about we name it “services”?

Reply:
Like 

subctl export services -n xxxx 

?

Reply:
Right, I wondered about that but hadn’t thought of a decent alternative — the risk with `export namespace` is that users will think that any changes made to the namespace afterwards will automatically be reflected.

Reply:
If people should be aware of that, by adding a new flag when using `export namespace` like `--include-submariner-services`.

Reply:
Which services are you referring to? Submariner doesn’t create services apart from the metrics services in `submariner-operator`.

Reply:
FWIR, we do create internal services for GN as well.  If we have to add complexity to exclude certain services, that kind of throws up a red flag for me as to whether we should provide this command. I’m not sure how common it would be for a user to blindly export all services in a namespace. If they really want to do that, they could easily write a simple script to do it. Also this would be a one-time export which would diminish its usefulness if a user expects it to be dynamic.

Reply:
Ah yes, the externalIP services...

Reply:
Agree with @tpantelis. Submariner creates (and owns) an internal service for every exported service in the same namespace where the original service resides. Even I feel that the requirement can be easily achieved with a simple script.